src/app/components/cells/MetaCellHeader.jsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+
+import PropTypes from "prop-types";
+import {
+  ContextMenuButton,
+  ContextMenu
+} from "../columns/ColumnHeaderFragments";
+import classNames from "classnames";
+
+class MetaCellHeader extends React.Component {
```

Comment:
Gibt es einen Grund, warum das eine Klassenkomponente ist? Und falls ja, könnte man nicht auch PureComponent verwenden?

Reply:
Nein, es gibt keinen Grund. Ich hatte die Komponente analog zu den Komponenten MetaCell und ColumnHeader implementiert. 
Mir war bis gerade eben gar nicht bewusst, dass die im Projekt verwendete React-Version doch nicht so alt ist wie gedacht. Ich werde die Komponente als Funtional-Component mit Hooks refactoren.

Reply:
Es gibt nur noch einige sehr alte Komponenten, die noch als Klassen implementiert sind und in zweien haben wir aus Performancegründen sehr viele Dinge auf React-falsche Art gecached, was nur durch wildes Hacken der Komponentenklassen ging.